### ISSUE TITLE
The file path displayed on the html file

### ISSUE THREAD

**Issue by ym-chen** [gagneurlab/drop#345]
Hi, 
    I find that the result file, displayed on the Output/html/Project_index.html, is used the absolute path. But the absolute path is the path on the server or cluster. If I download the Output directory to my personal computer or share to other member, these paths become wrong.
    I want to change the absolute path to relative path. Can I set it on the config.yaml? Such as set the "indexWithFolderName: false" in config.yaml.

**Comment by nickhsmith** [comment]
Yes, this is a known issue. The underlying indexing tool we use `wbuild` is set up for absolute paths. But all of the underlying pages are there. If you navigate to `path/to/html/` you can see files such as `AberrantExpression/Outrider/v29/Summary_outrider.html` that should be fully formed.
